app/views/proposals/_proposal_form.html.haml

```diff
@@ -55,7 +55,11 @@
     %p.text-right
       = link_to '#description', 'data-toggle' => 'collapse' do
         Do you require something special?
-    .collapse#description
+  -if @event.description.blank? 
```

Comment:
I would try to avoid repeating the input line, something like:

``` css
#description{ class: "collapse#{ ' in' if @event.description.present? }" }
= f.input :description, input_html: { rows: 5 }, label: 'Requirements', placeholder: 'Eg. Whiteboard, printer, or something like that.'
```